### PR TITLE
Improve product image management

### PR DESCRIPTION
**Pull request by Paphos**
- Add 'Title' field in 'Edit product' > 'Images' tab
- Add PREVIOUS and NEXT buttons when editing a product image
- Allow to select and delete multiple images in 'Product editing'
screenshot_image_management

### CODE REVIEW

**Comment by roadster31** [comment]
The image title could be modified from the image list page, and this is a great idea. However, there is no way to select the current language from the image list page. Additionally, the current editing language is not displayed on the image list page.

Could you consider integrating the [standard toolbar](https://github.com/thelia/thelia/blob/master/templates/backOffice/default/includes/inner-form-toolbar.html) (without currencies and buttons) to provide a way to select edition language and display current edition language ?

File: templates/backOffice/default/includes/image-upload-form.html [hunk 1]
```diff
@@ -12,7 +12,25 @@
 
 {hook name="tab-image.top" id=$parentId type=$imageType}
 
-<div class="image-manager" >
+<div class="image-manager form-container" >
+
+    {include
+        file = "includes/inner-form-toolbar.html"
```
Comment (Paphos) [inline templates/backOffice/default/includes/image-upload-form.html]: Isn't it what I've done already ?

Comment (roadster31) [reply templates/backOffice/default/includes/image-upload-form.html]: Sorry, I didn't see it. This is fine :)

**Comment by roadster31** [comment]
When the language is changed in the Image tab, the Genereral tab is displayed instead o the Images tab.

**Comment by roadster31** [comment]
When testing this PR on the master branch, I've experienced some problems.

1) Wrong tab is  displayed on language change :
- Open a folder edit page.
- Click the Image tab
- Change the language by clicking one of the toolbar flags
- The page is reloaded, and the General tab is displayed instead of the Image tab.

2) Changing the image title from the title field on the Image tab is not working for something else than a product, as 'product' type is hard coded in the URL generation : https://github.com/thelia/thelia/pull/2445/files#diff-3b7af29abd03594232899932b7f38186R58

**Comment by Paphos** [comment]
Done! =)

**Comment by roadster31** [comment]
Something is still not working properly : when I click the green check on categories images, I'm redirected to the http://myshop.com/admin/catalog?current_tab=images URL

Could you please test the PR on categories, folders, contents, brands, etc. ?

**Comment by Paphos** [comment]
Well this is strange, I actually tested it on products, categories, folders and brands and I didn't have the bug and now it seems to occur more or less randomly ... I think I didn't clear the cache, that's why I didn't see it.
Anyway, I changed `{navigate to='previous'}` into `{navigate to='current'}` on my links and it seems to work fine with all the pages you mentionned.

**Comment by roadster31** [comment]
Changing titles seems to be fine now, thanks.

One more thing : when "Delete selected image" is clicked when no image is selected the following dialog is displayed: 

img-2017-11-03_11-55-53

**Comment by Paphos** [comment]
Fixed! 😄

**Comment by roadster31** [comment]
Wrong dialog type :

img-2017-11-09_13-00-04

**Comment by Paphos** [comment]
Fixed!

**Comment by roadster31** [comment]
Thanks. What about implementing the same function for documents ?

**Comment by Paphos** [comment]
Ok I'll do it in an other PR if I have time.

**Comment by roadster31** [comment]
Could you please rebase ?

**Comment by roadster31** [comment]
Thanks !

**Pull request merged**